### ISSUE TITLE
Color Contrast is Insufficient 

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#3415]
**Functionality:**
Import a Monitoring Plan

**Issue:**
Color contrast ratio between text and background is less than 4.5:1. This is a violation of HHS Guidelines and WCAG 2.0 Success Criterion 1.4.3. See image below:
![image.png](https://images.zenhubusercontent.com/610014e44c513c0bad9f811c/6f7a9eef-9da3-460b-b3d3-a800683b9781)

**Suggested Resolution:**
Change color scheme by lightening or darkening one or both colors. For further assistance, see: 
Free contrast analyzer tool 
W3 General Technique #18

**Comment by aprematta** [comment]
Testing complete, the issue has been fixed
